test/OpenTelemetry.Instrumentation.AWS.Tests/OpenTelemetry.Instrumentation.AWS.Tests.csproj

```diff
@@ -10,6 +10,10 @@
   <ItemGroup>
     <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.400" />
     <PackageReference Include="AWSSDK.S3" Version="3.7.400" />
+    <PackageReference Include="AWSSDK.Bedrock" Version="3.7.303.3" />
+    <PackageReference Include="AWSSDK.BedrockRuntime" Version="3.7.303.3" />
+    <PackageReference Include="AWSSDK.BedrockAgent" Version="3.7.303.3" />
+    <PackageReference Include="AWSSDK.BedrockAgentRuntime" Version="3.7.303.3" />
```

Comment:
In previous release you decided to support 3.7.400 and newer. Please update versions